ui/nuxeo-video/nuxeo-video-conversions.js

```diff
@@ -31,7 +32,7 @@ import '../widgets/nuxeo-tooltip.js';
   class VideoConversions extends mixinBehaviors([I18nBehavior, FormatBehavior], Nuxeo.Element) {
     static get template() {
       return html`
-        <style>
+        <style include="nuxeo-styles">
```

Comment:
isn't `nuxeo-styles` only defined in Web UI ?!

Reply:
yes good point :/ reverting back